### PR TITLE
Adds ammo bandoliers

### PR DESCRIPTION
**Pull request by mugling**
No doubt @Rivet-the-Zombie will have some further suggestions

### CODE REVIEW

File: data/json/items/armor/bandolier.json [hunk 1]
```diff
@@ -1,24 +1,112 @@
...
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "TORSO" ],
+    "coverage": 10,
+    "encumbrance": 6,
+    "material_thickness": 1,
+    "use_action": {
+        "type": "bandolier",
+        "capacity": 18,
+        "ammo": [ "32", "38", "44" ]
```
Comment (Rivet-the-Zombie) [inline data/json/items/armor/bandolier.json]: This should probably include other rounds used by revolvers like 9mm, .45, and 8x40mm.

File: data/json/items/armor/bandolier.json [hunk 2]
```diff
@@ -1,24 +1,112 @@
...
-    "covers": ["TORSO"],
+    "covers": [ "TORSO" ],
     "coverage": 10,
-    "encumbrance": 2,
+    "encumbrance": 6,
     "material_thickness": 1,
     "use_action": {
         "type": "bandolier",
         "capacity": 12,
         "ammo": [ "shot" ]
```
Comment (Rivet-the-Zombie) [inline data/json/items/armor/bandolier.json]: Maybe this could also carry 20x66mm rounds?

File: data/json/items/armor/bandolier.json [hunk 3]
```diff
@@ -1,24 +1,112 @@
...
+    "symbol": "[",
+    "color": "dark_gray",
+    "covers": [ "TORSO" ],
+    "coverage": 10,
+    "encumbrance": 6,
+    "material_thickness": 1,
+    "use_action": {
+        "type": "bandolier",
+        "capacity": 16,
+        "ammo": [ "22", "223", "3006", "308", "762", "762R"  ]
```
Comment (Rivet-the-Zombie) [inline data/json/items/armor/bandolier.json]: Maybe this could also carry 8x40mm?

**Comment by Coolthulhu** [comment]
Balancing: The torso encumbrance is "worth" more here, since torso is bigger and has more than double the effects of hand encumbrance. Wrist bandoliers should have higher encumbrance even at low capacity.

Does VARSIZE on torso bandoliers make sense? They look relatively loose from what I can tell.

**Comment by mugling** [comment]
> Wrist bandoliers should have higher encumbrance even at low capacity.

Fair enough, see 90090da

> Does VARSIZE on torso bandoliers make sense?

I would suppose it represents adjusting it to be the correct length and such especially considering  they are also `OVERSIZE`.

**Comment by Coolthulhu** [comment]
Reloading from bandoliers is relatively annoying:
* RELOAD_ONE requires picking the source every time
* Keybinds aren't assigned properly. For example, when I equipped 2 bandoliers with 8x40 mm ammo and had a stack of same ammo in the inventory, all 3 sources got 'm' keybind. This also means that 'r'->'r', which should reload using the default option, did nothing.
* Reloading the bandolier itself requires activating it. There is no automatic way of distributing ammo to bandoliers/magazines etc.

**Comment by mugling** [comment]
> RELOAD_ONE requires picking the source every time

Can you open an issue for this and I'll look into it

> Keybinds aren't assigned properly

This is part of a wider issue with invlets and has been reported before

> Reloading the bandolier itself requires activating it

We could allow them to reload directly (like ammo pouches?)

> There is no automatic way of distributing ammo to bandoliers/magazines etc.

Can you explain more?

**Comment by mugling** [comment]
> This is part of a wider issue with invlets and has been reported before

See #15455

**Comment by Coolthulhu** [comment]
> Can you open an issue for this and I'll look into it

Opened #15702

> We could allow them to reload directly (like ammo pouches?)

With current usage cases, it wouldn't help a lot, but it would be consistent and would work as expected, which is a plus.

> Can you explain more?

I was thinking something like "reload guns and accessories" long action, which would start by reloading all guns, then all magazines, then all pouches and bandoliers. Could ask for ammo, but should remember the choices (ie. if an ammo stack is selected for shotgun reload, it doesn't ask per cartridge).

Finding a binding for that could be a problem.

**Pull request merged**